skimage/transform/_geometric.py

```diff
@@ -1244,20 +1330,27 @@ class EuclideanTransform(ProjectiveTransform):
     transformation is only a translation, you may use the implicit parameter
     `translation`; otherwise, you must use `matrix`.
 
+    The implicit parameters are applied in the following order:
+
+    1. Rotation;
+    2. Translation.
+
     Parameters
     ----------
     matrix : (D+1, D+1) array_like, optional
         Homogeneous transformation matrix.
     rotation : float or sequence of float, optional
-        Rotation angle, clockwise, as radians. If given as
-        a vector, it is interpreted as Euler rotation angles [1]_. Only 2D
-        (single rotation) and 3D (Euler rotations) values are supported. For
-        higher dimensions, you must provide or estimate the transformation
-        matrix.
+        Rotation angle, clockwise, in radians. If given as a vector, it is
+        interpreted as Euler rotation angles [1]_. Only 2D (single rotation)
+        and 3D (Euler rotations) values are supported. For higher dimensions,
+        you must provide or estimate the transformation matrix instead, and
+        pass that as `matrix` above.
     translation : (x, y[, z, ...]) sequence of float, length D, optional
         Translation parameters for each axis.
     dimensionality : int, optional
-        The dimensionality of the transform.
+        Fallback number of dimensions for transform when no other paremeters
+        are specified.  Otherwise ignored, and we infer dimensionality from the
```

Comment:
```suggestion
        Fallback number of dimensions for transform when no other parameter
        is specified.  Otherwise ignored, and we infer dimensionality from the
```
typo + singular (thinking, there's not even a single other parameter)